.github/labeler.yml

```
@@ -0,0 +1,13 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/clx/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/clx/labels
+
+Python:
+  - 'python/**'
+  - 'notebooks/**'
+
+ci:
```

```suggestion
gpuCI:
```
there's no `ci` label, so this can probably be changed to `gpuCI`.